examples/hello-world-fetch.rs

```diff
@@ -56,6 +56,8 @@ async fn main() -> Result<()> {
         "node relay server url: {:?}",
         node.endpoint()
             .home_relay()
+            .initialized()
+            .await
             .expect("a default relay url should be provided")
             .to_string()
```

Comment:
```suggestion
        node.endpoint()
            .home_relay()
            .get()?
            .expect("a default relay url should be provided")
            .to_string()
```

Reply:
Or, alternatively only use `node.endpoint().home_relay().initialized().await?.to_string()` without the `expect`.

Reply:
fixed